src/news-feed.js

```diff
@@ -91,8 +90,8 @@ export default class NewsFeed {
 
         // Figure out time ranges to consider resonable
         const now = dayjs()
-        const yearAgo = dayjs().subtract(1, 'year')
-        const nextMonth = dayjs().add(1, 'month')
+        const yearAgo = now.subtract(1, 'year')
+        const nextMonth = now.add(1, 'month')
```

Comment:
Is this going to result in yearAgo and nextMonth being equal. Check how the immutability works on this, as I am not sure this will result in the outcome you think it will.

Reply:
rookie mistake (not reading docs 🤦 ) thank you for spotting 🙏